ntc_templates/templates/cisco_ios_show_interfaces.textfsm

```diff
@@ -9,7 +9,7 @@ Value IP_ADDRESS (\d+\.\d+\.\d+\.\d+\/\d+)
 Value MTU (\d+)
 Value DUPLEX (([Ff]ull|[Aa]uto|[Hh]alf|[Aa]-).*?)
 Value SPEED (.*?)
-Value MEDIA_TYPE (\S+(\s+\S+)?)
+Value MEDIA_TYPE (\S+[\s\S]*)
```

Comment:
What is this addressing? Can you add a test for that? 

Additionally, `\s` == white space and `\S` == non white space, I think can simply be 

`(\S+.*)`

Reply:
The value of media type can contain strings of characters alternated with spaces. I added some tests about this.